fv3core/examples/standalone/build_scripts/build_example.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+# Example bash script to build Pace to run locally outside of a container, assuming Pace is cloned into $HOME
+
+set -e -x
+
+# module load necessary system software
+module load gcc/11.3.0
+module load openmpi
+module load cuda/10.2
+module load python/3.8.13
```

Comment:
Could you specify a particular system and add a date for the last time this was updated, so that readers can get a better idea of how applicable it is?